src/components/shared/__tests__/Validations.spec.tsx

```diff
@@ -127,5 +127,17 @@ describe('Validations component', () => {
 
       expect(wrapper).toHaveTextContent('>= 0');
     });
+
```

Comment:
as a part of the walkthrough .. the test case needs to be recorrected to identify the location of the min value and max value.  This test case is passing without the fix as well.

Reply:
Hi Prafulla, I have removed the testcase as this issue is not reproducible in json-schema-viewer and for platform internal there will be a playwright testcase.